server/historian/packages/historian-base/src/services/redisTenantCache.ts

```diff
@@ -42,19 +46,29 @@ export class RedisTenantCache {
 		value: string = "",
 		expireAfterSeconds: number = this.expireAfterSeconds,
 	): Promise<void> {
-		const result = await this.client.set(this.getKey(key), value, "EX", expireAfterSeconds);
+		const result = await this.redisClientConnectionManager
+			.getRedisClient()
+			.set(this.getKey(key), value, "EX", expireAfterSeconds);
 		if (result !== "OK") {
 			throw new Error(result);
 		}
 	}
 
 	public async delete(key: string): Promise<boolean> {
-		const result = await this.client.del(this.getKey(key));
-		return result === 1;
+		try {
+			const result = await this.redisClientConnectionManager
+				.getRedisClient()
+				.del(this.getKey(key));
+			return result === 1;
+		} catch (error) {
```

Comment:
let's add this try catch logic in all the other methods here as well.

Reply:
The methods calling the catch methods had a try-catch block. Hence, I did not add try-catch for the other methods. But for consistency, I can add try-catches.